PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+#### What does this PR do?
+#### Do you have any concerns with this PR?
+#### How can the reviewer verify this PR?
+#### Any background context you want to provide?
+#### Screenshots (if appropriate)
```

Comment:
Sorry...one more change.  We are much more likely to have logs than screenshots.

Screenshots or logs (if appropriate)

Reply:
Roger that. Good call. I'll add that.